### DISCUSSION TITLE
How does --viz-namespace works?

### DISCUSSION THREAD

**Discussion by Cobollatin** [linkerd/linkerd2#11776] [Help] [answered]
I am using this command to install viz, I installed the control pane and linkerd in the same namespace, however when I try to check viz I get This is not a linkerd-viz namespace error, what am I doing wrong? 

helm install linkerd-viz linkerd/linkerd-viz `
    --set linkerdNamespace=apps-system `
    --set tapInjector.namespaceSelector=apps-system `
    --create-namespace `
    --namespace apps-system
 

![image](https://github.com/linkerd/linkerd2/assets/40482066/507d8faf-fe3b-47c9-8fd0-b684c6de5b7c)

Linkerd check works
![image](https://github.com/linkerd/linkerd2/assets/40482066/2c4ed9a1-cc4b-43e5-9fae-336e764aa340)
![image](https://github.com/linkerd/linkerd2/assets/40482066/97656216-c210-4d37-bafe-35216d5aeb86)

**Comment by alpeb** [comment, marked as answer]
Hi, it's not supported to share the core linkerd namespace with extensions'. Each extension needs to go into its own separate namespace, otherwise you run into situations like this.
